tools/workspace-plugin/src/generators/eslint-rule/generator.spec.ts

```diff
@@ -68,12 +68,10 @@ describe('eslint-rule generator', () => {
     `);
 
     expect(tree.read(paths.spec, 'utf-8')).toMatchInlineSnapshot(`
-      "import { TSESLint } from '@typescript-eslint/utils';
+      "import { RuleTester } from '@typescript-eslint/rule-tester';
       import { rule, RULE_NAME } from './uppercase';
 
-      const ruleTester = new TSESLint.RuleTester({
-        parser: require.resolve('@typescript-eslint/parser'),
-      });
+      const ruleTester = new RuleTester();
```

Comment:
what about the parser ? will the new package set ts-eslint parser by default ? if not please keep the parser setting

Reply:
Checked this before removal, in docs mentioned, that it's a default parser, so I guess it's ok :) 
![Screenshot 2024-07-24 at 16 27 02](https://github.com/user-attachments/assets/cef7be59-cf88-4a99-a697-459ad88844f0)